types/detect-it/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Don't need this file. You are depending on the `@types` for these, not on the packages theirselves. We will automatically detect `@types` dependencies, so you don't need to specify those anywhere.